src/main/scala/treadle/executable/RenderedExpression.scala

```diff
@@ -135,9 +135,9 @@ class ExpressionViewRenderer(
               symbolTable.inputPortsNames.contains(symbol.name) ||
               symbolsSeen.contains(symbol)
             )) {
-            if(showValues) {
-              symbolsToDo.enqueue(SymbolAtDepth(symbol, displayDepth + 1, dataTime, dataArrays))
-            }
+//            if(showValues) {
```

Comment:
Remove commented out code?
```suggestion
//            if(showValues) {
 ...
 //           }
```

Reply:
comments removed